app/helpers/schools/dashboards_helper.rb

```diff
@@ -13,4 +13,14 @@ def numbered_circle(number, id: nil, colour: 'red', width: 26, height: 30, font_
   def school_enabled_description(school)
     school.enabled? ? "enabled" : "disabled"
   end
+
+  def show_no_placement_dates_warning?
+    @current_school.availability_preference_fixed? &&
+      @current_school.bookings_placement_dates.none?
```

Comment:
should this be `bookings_placement_dates.future.none?` ?

Reply:
Good call